admin-dev/themes/new-theme/js/pages/import-data/ImportDataTable.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+const $ = window.$;
+
+const $importDataTable = $('.js-import-data-table');
+
+/**
+ * Pagination directions - forward and backwards.
+ */
+const FORWARD = 'forward';
+const BACKWARDS = 'backwards';
```

Comment:
why BACKWARD**S**? i suggest BACKWARDS/FORWARDS or BACKWARD/FORWARD, seems more consistent that way?

Reply:
yeah nice catch, I thought that's how it is in English, but apparently both options are valid (one is british, one is american). I'll unify it :)